stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/imagepreview/ImagePreviewActivity.kt

```diff
@@ -69,8 +69,6 @@ public class ImagePreviewActivity : AppCompatActivity() {
             finish()
```

Comment:
Maybe we should throw or log an error that there is no `messageId` to view these images.